highway

```diff
@@ -0,0 +1,199 @@
+  echo -e "${Yellow}docker-compose.highway.yml downloaded successfully.${NC}"
+}
+
+function pullImages {
+  ${docker_compose} pull
+}
+
+
+if [ ! -f $(pwd)/strato ]; then
+    echo -e "${Red}Should be run from within the strato-getting-started directory. Exit.${NC}"
+    exit 4
+fi
+
+if ! docker ps &> /dev/null
+then
+    echo -e "${Red}Error: docker is required: https://www.docker.com/ . If you have it installed - you may need to execute as super user${NC}"
+    exit 1
+fi
+
+if ! docker-compose -v &> /dev/null
+then
+    echo -e "${Red}Error: docker-compose is required: https://docs.docker.com/compose/install/"
+    exit 2
+else
+  if ! docker compose version &> /dev/null
+  then
+    docker_compose="docker-compose -p highway -f docker-compose.highway.yml --log-level ERROR"
+  else
+    docker_compose="docker compose -p highway -f docker-compose.highway.yml"
+  fi
+fi
+
+STRATOGS_REQUIRED_VERSION=$(grep strato-getting-started-min-version docker-compose.highway.yml | awk -F":" '{print $NF}')
+if [ ${STRATOGS_REQUIRED_VERSION} ]; then
+  STRATOGS_CURRENT_VERSION=$(< VERSION)
+  if ! awk -v VER=${STRATOGS_CURRENT_VERSION//.} -v REQ_VER=${STRATOGS_REQUIRED_VERSION//.} 'BEGIN {exit (VER < REQ_VER)}'
+  then
+      echo -e "${Red}The STRATO version from docker-compose.highway.yml is incompatible with this strato-getting-started. Please update to v${STRATOGS_REQUIRED_VERSION}."
+      exit 12
+  fi
+fi
+
+while [ ${#} -gt 0 ]; do
+  case "${1}" in
+  --help|-h)
+    help
+    exit 0
+    ;;
+  --stop)
+    stop
+    exit 0
+    ;;
+  --start)
+    start
+    exit 0
+    ;;
+  --down)
+    down
+    exit 0
+    ;;
+  --wipe)
+    wipe
+    exit 0
+    ;;
+  --compose)
+    getCompose
+    exit 0
+    ;;
+  --pull)
+    pullImages
+    exit 0
+    ;;
+  *)
+    echo -e "${Red}Unknown flag ${1} provided, please check --help. Exit.${NC}"
+    exit 5
+    ;;
+  esac
+  shift 1
+done
+
+outputLogo
+
+if [ "$ssl" = true ] ; then
+  http_protocol=https
+  main_port=${HTTPS_PORT}
+else
+  http_protocol=http
+  main_port=${HTTP_PORT}
+fi
+export sslCertFileType=${sslCertFileType:-pem}
+
+echo "" && echo "*** Common Config ***"
+echo "HTTP_PORT: $HTTP_PORT"
+echo "HTTPS_PORT: $HTTPS_PORT"
+echo "ssl: $ssl"
+echo "sslCertFileType: $sslCertFileType"
+
+if [ ! -f docker-compose.highway.yml ]
+then
+  getCompose
+else
+  echo -e "${BYellow}Using the existing docker-compose.highway.yml (to download the most recent stable version - remove the file and restart the script)${NC}"
+fi
+
+# COMPOSE FILE PRE-PROCESSING
+function cleanup {
+  rm docker-compose-highway-temp.yml
+}
+trap cleanup EXIT
+
+if [ "$ssl" != true ]; then
+  sed -n '/#TAG_REMOVE_WHEN_NO_SSL/!p' docker-compose.highway.yml > docker-compose-highway-temp.yml
+else
+  if [ "$HTTPS_PORT" != "8093" ]; then
```

Comment:
we should change this port to smth else, as the 8093 is taken by the Vault. But actually, I think 443 should be the default for Highway.